deployments/follow.go

```diff
@@ -51,8 +51,12 @@ func Stream(opts *StreamOpts) error {
 	}
 
 	var event deployEvent
-	var currentDeployment *scalingo.Deployment
-	oldStatus := ""
+	currentDeployment := &scalingo.Deployment{
+		ID: opts.DeploymentID,
+	}
+	anyDeployment := currentDeployment.ID == ""
```

Comment:
I don't really get the meaning of `anyDeployment`.
Moreover, maybe adding a bit of comments would help understand what happens here. In which context the `DeploymentID` is not provided?

Reply:
When you run the command `scalingo deployment-follow` which display every events about all the deployments of an app